packages/dds/tree/src/test/shared-tree-core/sharedTreeCore.spec.ts

```diff
@@ -202,38 +174,38 @@ describe("SharedTreeCore", () => {
 		// Additionally, by policy, the base commit of the trunk is never evicted, which adds another "off by one".
 		//
 		//                                            trunk: [seqNum1, (branchBaseA, branchBaseB, ...), seqNum2, ...]
-		changeTree(tree);
+		changeTree(tree.kernel);
```

Comment:
It might make more sense to update `changeTree` rather than update all the call sites.

Reply:
Generally I think this test file should unit test SharedTreeCore and not TestSharedTreeCore so we should work toward doing everything in terms of SharedTreeCore, and not in terms of a wrapper around it. I have made some changes to better reflect this.